views/cart.html

```diff
@@ -0,0 +1,257 @@
+                        <span class="label-text label-text-primary">Restaurants</span>
+                    </li>
+                </a>
+                <a href="../views/cart.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">person</span>
+                        <span class="label-text label-text-primary">cart</span>
+                    </li>
+                </a>
+                <a href="../views/wishlist.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">queue</span>
+                        <span class="label-text label-text-primary">wishlist</span>
+                    </li>
+                </a>
+                <a href="../views/login.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">logout</span>
+                        <span class="label-text label-text-primary">Logout</span>
+                    </li>
+                </a>
+            </ul>
+        </div>
+
+    </nav>  
+
+    <main class="mb-3">
+       <section class="hotel-lists gap pt-1">
+           <div class="menu-lists">
+               <h1>Checkout</h1>
+            <div class="menu-section mb-1 mt-1">
+                <figure class="ui-card-default py-1 width-full">
+                    <blockquote class="flex flex-wrap gap pb-1">
+                      <p class="font-bold font-sm ">Mehul Satardekar</p>
+                      <span class="col-gray">(mehulsatardekar@gmail.com)</span>
+                    </blockquote>
+                    <figcaption class="blockquote-footer font-mid-light">
+                      <cite title="Source Title col-gray">You are securely logged in</cite>
+                    </figcaption>
+                  </figure>
+            </div>
+
+            <div class=" card-py gap shadow-sm mb-1 flex flex-column  pb-3">
+                
+                <div class="flex flex-between flex-wrap flex-align-item-center">
+                    <h1 class="font-mid-bold">Delivery Address</h1>
+                    <button class="btn-outline-primary btn btn-sm">Change</button>
+
+                </div>
+                <div class="flex flex-column gap-1">
+                    <span class="font-md-light">Deliver to:  <span class="label-text">John Doe, 560078</span></span>
+                    <span class="label-text">#1/4 , 100ft Ring Road, Jp Nagar - 4 Phase, Dollars Colony, Bangalore</span>
+
+                </div>
+
+            </div>
+
+           <div class="ui-card-default py">
```

Comment:
use better class name instead of py

Reply:
py describe padding to all btw these naming conventions are being followed by a lot of CSS frameworks